test/xpu/run_test_with_skip.py

```diff
@@ -790,6 +790,16 @@ def launch_test(test_case, skip_list=None, exe_list=None):
     # 2. Half dtype is a common dtype in workloads.
     # So far CUDA doesn't support Half, so that XPU fails as we aligned claimed dtypes with CUDA in test infra.
     "test_dtypes_nextafter_xpu",
+
+    # Deselect temporarily
```

Comment:
Please elaborate the justification.

Reply:
Done.

Reply:
![test_dtypes_fft](https://github.com/user-attachments/assets/c9354806-d9c7-43fc-b26e-67a15d7fbab5)
At present, only two `test_dtypes` cases are reserved for fft.